store/plugins/preferencesSyncPlugin.ts

```diff
@@ -0,0 +1,91 @@
+import { DisplayPreferencesDto } from '@jellyfin/client-axios';
+import { Plugin } from 'vuex';
+import { AppState } from '~/store';
+import { syncedMutations } from '~/store/clientSettings';
+
+export const preferencesSync: Plugin<AppState> = (store) => {
+  store.subscribe(async (mutation) => {
+    if (
+      syncedMutations.some((syncedMutation) =>
+        mutation.type.includes(syncedMutation)
+      ) &&
+      store.$auth.loggedIn
```

Comment:
Why not directly use the `this.$auth`?

Reply:
You don't get the Nuxt context as `this` in the Vuex plugins. You get them through the store.

Reply:
My bad didn't see it. I thought it was a Nuxt one :p